client/shipping-label/state/actions.js

```diff
@@ -140,21 +140,41 @@ const refreshPreview = ( dispatch, getState, context ) => {
 	} );
 };
 
-export const openPrintingFlow = () => ( dispatch, getState, context ) => {
+export const openPrintingFlow = () => (
+	dispatch,
+	getState,
+	context,
+	getErrors = getFormErrors
```

Comment:
This method didn't receive a `getErrors` param before, now it can receive one. If it doesn't, it uses `getFormErrors`, so it works as before. **But**, in the tests, this method can be called with a custom `getErrors` function, which makes it easier to "mock" a validator.

That's why I brought this to attention: this is code that only serves the tests. If there's a less intrusive way to mock the validator, we should do that instead.